consensus/types.go

```diff
@@ -183,11 +182,11 @@ type CoveredFields struct {
 // TODO: Switch to a different constant because of using 128 bit values for the
 // currency.
 func CalculateCoinbase(height BlockHeight) Currency {
-	if Currency(height) >= InitialCoinbase-MinimumCoinbase {
-		return MinimumCoinbase * 100000
-	} else {
-		return (InitialCoinbase - Currency(height)) * 100000
+	base := InitialCoinbase - uint64(height)
+	if base < MinimumCoinbase {
+		base = MinimumCoinbase
 	}
+	return NewCurrency(base * 1e5)
```

Comment:
https://github.com/NebulousLabs/Sia-UI/issues/9

I was sort of thinking that `1e12` would be one sia. `1 << 80` would be 1.2 Tsia. The first block would have 300Psia. `1 << 0` would be 1 psia.

Most people would probably talk in Tsia under this method. But it's stil confusing and could probably use some work.